client/app/views/index.js

```diff
@@ -24,11 +24,17 @@ function fetchPipelineIds() {
 }
 
 function appendPipeline(pipeline) {
-  dom('#content-container').append(renderPipeline(pipeline));
+  var pipelineId = pipeline.name.toLowerCase().replace(' ', '-');
+  var pipelineElement = dom('#' + pipelineId);
+  if (pipelineElement.exists()) {
+    pipelineElement.replace(renderPipeline(pipeline));
+  } else {
+    dom('#content-container').append(renderPipeline(pipeline));
+  }
 }
 
 function fetchPipeline(id) {
-  var errorMsg = 'Fetching Pipeline "'+id+'" failed. Please refresh the page.';
+  var errorMsg = 'Fetching Pipeline "' + id + '" failed. Please refresh the page.';
```

Comment:
I'm not with my laptop until Saturday night but as soon as I'm back I'm
happily rejec... cof cof cof... reviewing your PR =D

PS: I'm glad the pipeline is back =D
Em 31 de mar de 2016 13:28, "Hound" <notifications@github.com> escreveu:

> In client/app/views/index.js
> <https://github.com/jenkins-pipeline/jenkins-pipeline/pull/77#discussion_r58084617>
> :
>
> >  }
> >
> >  function fetchPipeline(id) {
> > -  var errorMsg = 'Fetching Pipeline "'+id+'" failed. Please refresh the page.';
> > +  var errorMsg = 'Fetching Pipeline "' + id + '" failed. Please refresh the page.';
>
> Line is too long.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-pipeline/jenkins-pipeline/pull/77/files/782cdb3c53474f7f2572da3a4cf3300b983a20b6#r58084617>
>